engine/script/src/script.cpp

```diff
@@ -1820,7 +1820,7 @@ namespace dmScript
     // to calculate the length of the input string or output string
     static char* ConcatString(char* w_ptr, const char* w_ptr_end, const char* str)
     {
-        while ((w_ptr != w_ptr_end) && *str)
+        while ((w_ptr != w_ptr_end) && str && *str)
```

Comment:
As I see the only thing may be cause of the crash in this line is deref of the str  if it’s `NULL`
->
Exception Type: Unknown (SIGSEGV))

So this while should help

Reply:
But, this is still just guesswork?

Reply:
sure, we don't have a repo case, but we know the crash happens in this particular line